static/app/data/experimentConfig.tsx

```diff
@@ -12,22 +12,16 @@ export const unassignedValue = -1;
  * Frontend experiment configuration object
  */
 export const experimentList: {
-  assignments: number[];
+  assignments: string[];
```

Comment:
@priscilawebdev can we keep `number[]` for now since I'm adding an experiment that uses it? Though @malwilley is going to work on making sure this doesn't need an explicit type here and instead use `as const` which is how this is supposed to work

Reply:
ok so if I use `[0,1,2]` will this work with the changes I pushed in my `getsentry` pr? you mentioned that they would be incompatible and I would have to create another PR to update it... I prefer it to be explicit

Reply:
@priscilawebdev `number[]  | string[]` is fine for now

Reply:
done!